Activity.js

```diff
@@ -6,12 +6,10 @@ class Activity{
         this.seconds = seconds;
         this.completed = false;
         this.id = `ID${Date.now()}`;
-        // this.countdownTimeInMS = (this.minutes*60000)+(this.seconds*1000);
-        // this.duration = countdownTimeInMS/1000
         this.running = false
     }
 
-    startTimer(){
+    startTimer(event){
```

Comment:
I dont think we need the event parameter here as we aren't using it anywhere else in the function.

Reply:
deleted event param!